core/capabilities/targets/write_target.go

```diff
@@ -232,11 +248,12 @@ func (cap *WriteTarget) Execute(ctx context.Context, rawRequest capabilities.Cap
 		cap.lggr.Infow("returning without a tranmission attempt - transmission already attempted, receiver was marked as invalid", "executionID", request.Metadata.WorkflowExecutionID)
 		return success(), nil
 	case transmissionInfo.State == 3: // FAILED
-		if transmissionInfo.GasLimit.Uint64() > cap.receiverGasMinimum {
-			cap.lggr.Infow("returning without a tranmission attempt - transmission already attempted and failed, sufficient gas was provided", "executionID", request.Metadata.WorkflowExecutionID, "receiverGasMinimum", cap.receiverGasMinimum, "transmissionGasLimit", transmissionInfo.GasLimit)
+		receiverGasMinimum := request.Inputs.GasLimit - FORWARDER_CONTRACT_LOGIC_GAS_COST
```

Comment:
```suggestion
		receiverGasMinimum := request.Inputs.GasLimit + FORWARDER_CONTRACT_LOGIC_GAS_COST
```

Users shouldn't have to account for contract logic gas cost and only specify what their contract needs.

Reply:
what you say makes sense, but just to confirm, this is a change to the original logic, the reason for this is because of the change in how the limit is specified.

Reply:
Sorry, you are right, my comment is incorrect. It should be `-` 🙈

Reply:
as per slack chat, the sign should be kept as negative